static_report/src/components/shared/Tables/TableList/CRTableListItem/index.tsx

```diff
@@ -53,17 +61,21 @@ export function CRTableListItem({
             />
           </Flex>
         </GridItem>
-        <GridItem>{children}</GridItem>
         <GridItem>
-          {isExpanded && (
-            <Flex
-              as="a"
-              data-cy="cr-navigate-report-detail"
-              onClick={() => onSelect()}
-            >
-              <Icon as={FiChevronRight} color="piperider.500" boxSize={6} />
-            </Flex>
-          )}
+          <Flex gap={2}>
+            {children}
+            {isExpanded && (
+              <ChakraLink
```

Comment:
We shouldn't hard-code the url link here, as it's not a page-level component.
Else if cloud uses diff route implementation, this won't work

Reply:
Yes, it should lift up to page level.

But I'm thinking about users experiment, because using `onSelect` cannot display the URL when users into, e.g.

| display URL | without URL|
| --- | --- |
| <img width="1266" alt="截圖 2022-09-08 10 49 04" src="https://user-images.githubusercontent.com/10325111/189023297-bc4dedc4-a3eb-47b5-9541-36cdc7be9f73.png"> | <img width="1261" alt="截圖 2022-09-08 10 49 11" src="https://user-images.githubusercontent.com/10325111/189023306-bbe1ce34-ff0d-4fc9-afa0-499a9f79c2a6.png"> |

What if I change implementation like:

```jsx
<SRTableListItem
    isExpanded={isExpanded}
    tableDatum={table}
>
  <Link href="/xxx">Go</Link>
</SRTableListItem>
```

We can implement route logic on the page level, what do you think?

Reply:
Hmm, this one is a tricky one. I'm thinking about the tradeoffs.

Personally, I'd prefer to keep this as an event emitter, as you can do alot more things on implementation's end (e.g. respond and setState where need be).
  
Supporting transclusion e.g. (nesting children inside) would mean allowing for some unintended use cases, and has a highly specific placement of the `{children}` (e.g. what if we want to wrap the entire item vs just that chevron-right on that specific div tag? Is this doable with `ReactNode` ?).

Alternatively, if the goal is to achieve an anchor tip (bottom-left tooltip of url), I think that maybe we should have a shareable dynamic wrapper for different `*Item` components. So that if some prop `{ MyWrapper: ComponentType }` is provided, it wraps a component's selectable area, while still exposing the event handler for upstream uses. 

thoughts?

Reply:
I think this PR purpose is to eliminate duplicate parts, these changes are out of scope, and I will revert them.

I agree with your points, if we want to do it, we can discuss it in the future.

> if the goal is to achieve an anchor tip (bottom-left tooltip of URL)
By the way, the goal is not only to display a *tip*, users can open another tab (right click) to visit the page.